app/assets/javascripts/archived_patients.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Can ditch the coffee and the scss file and the helper since we don't have the controller anymore, I think?